<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oracle_table_wasted_window</class>
 <widget class="QMainWindow" name="oracle_table_wasted_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oracle table wasted</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/MGrep.ico</normaloff>:/icons/icons/MGrep.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item row="3" column="2">
     <widget class="QLabel" name="l_search">
      <property name="text">
       <string>Table to search:</string>
      </property>
      <property name="buddy">
       <cstring>e_search</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="e_search"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="e_server_name"/>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="b_search">
      <property name="text">
       <string>Search</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/go.gif</normaloff>:/icons/icons/go.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="l_server_name">
      <property name="text">
       <string>Oracle name server:</string>
      </property>
      <property name="buddy">
       <cstring>e_server_name</cstring>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="5">
     <widget class="QLabel" name="l_list_label">
      <property name="text">
       <string>List of tables space wasted:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="20" column="0" colspan="5">
     <widget class="QTextEdit" name="e_sql_script">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>20</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="5">
     <widget class="QTableView" name="o_lst1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>50</verstretch>
       </sizepolicy>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="5">
     <widget class="QPushButton" name="b_create_script">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1) Select a table space&lt;/p&gt;&lt;p&gt;2) Click this button&lt;/p&gt;&lt;p&gt;3) The script is composed in the text box below, with a new numeric part&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Create script for optimizing the selected table</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/sql.gif</normaloff>:/icons/icons/sql.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="l_total_space_wasted">
      <property name="text">
       <string>Total space wasted GByte:</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLabel" name="l_total_space_wasted2">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>o_lst1</tabstop>
  <tabstop>b_create_script</tabstop>
  <tabstop>e_sql_script</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b_create_script</sender>
   <signal>clicked()</signal>
   <receiver>oracle_table_wasted_window</receiver>
   <slot>slot_create_script()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_search</sender>
   <signal>clicked()</signal>
   <receiver>oracle_table_wasted_window</receiver>
   <slot>slot_start_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_create_script()</slot>
  <slot>slot_start_search()</slot>
 </slots>
</ui>
